test_framework/test_assembler.py

```diff
@@ -27,8 +28,8 @@ def check_datafile(filename):
     assert len(bin_result) % 8 == 0
     assert len(bin_result) / 8 == len(data['raw'])
 
-    for i in xrange(0, len(bin_result), 8):
-        j = i/8
+    for i in six.moves.xrange(0, len(bin_result), 8):
```

Comment:
Wouldn't `from past.builtins import xrange` be enough here?

Reply:
Doesn't [past](https://pypi.org/project/past/) add another dependency? I changed the code so that it used try-except to define xrange in python3.